app/api/dao/mentorship_relation.py

```diff
@@ -159,6 +197,15 @@ def accept_request(user_id, request_id):
 
     @staticmethod
     def reject_request(user_id, request_id):
+        """Allows a given user to reject a particular request.
```

Comment:
```suggestion
        """Rejects a mentorship request.
```

Add a longer description 1 line below:
```
Reject mentorship request if the current user if is the receiver of the request.
```